### ISSUE TITLE
[Bug] Play/Pause, and skip buttons won't disappear

### ISSUE THREAD

**Issue by VetheonGames** [revanced/revanced-patches#929]
### Type

Other

### Bug description

On Samsung Galaxy Note 20 Ultra, the YouTube ReVanced video player constantly displays the play/pause and skip buttons and they are impossible to make disappear. They also darken the video when they are displayed, which makes the video virtually unwatchable.

### Steps to reproduce

Step1. Launch YouTube ReVanced on Samsung Galaxy Note 20 Ultra

Step2. Open a few videos

Step3. Eventually you'll encounter the issue

(I'm using basically every patch aside from Premium Branding, HDR, Tablet Viewer, and logs)

### Relevant log output

```shell
I wouldn't know how to get a log for this issue
```

### Screenshots or videos

![Screenshot_20221031-011121](https://user-images.githubusercontent.com/76892542/198951671-cb1bf60d-afc1-49f1-8601-5a2a0609ba3d.jpg)

### Additional context

I've tried uninstalling, repatching without the custom theme, and nothing changed.

I've tried patching a lower version of YouTube than the recommended and still no dice.

### Acknowledgements

- [ ] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
#338

**Comment by VetheonGames** [comment]
Hey giving an update here as I think I found a bug separate from the one noted above.

The buttons pop up and refuses to leave when SponsorBlock does an automatic skip.

Otherwise everything works to expectation.

To workaround the bug right now, I simply disabled SponsorBlock Automatic Skip.

I think this issue should be reopened if there isn't another open about it already. Unless I should report this to SponsorBlock instead of here..?
